script.js

```diff
@@ -1,4 +1,68 @@
+// initialise first mode asking input
+var mode = 'need input';
+
+// two dice array for ramdom dice roll
+var diceOne = [];
```

Comment:
I'm confused. Why do each of these dice need to be stored in an array? Doesn't each dice just have 1 value?